products/models.py

```diff
@@ -13,7 +13,7 @@ def __str__(self):
 		return self.name
 
 # Create your models here.
-class Stock(models.Model):
+class Products(models.Model):
```

Comment:
Product

Reply:
okay ill change products model to product